server_admin/topics/identity-broker/first-login-flow.adoc

```diff
@@ -64,3 +64,11 @@ Verify Existing Account By Re-authentication::
   User can also re-authenticate with some different identity provider, which is already linked to his {project_name} account.
   You can also force users to use OTP. Otherwise it's optional and used only if OTP is already set for the user account.
 
+==== Automatically Link Existing First Login Flow
+In order to configure an alternative first login flow in which users are automatically linked without prompt create a new flow with the following two authenticators:
+
+Create User If Unique::
+  This authenticator ensures unique users are handled.  Set the authenticator requirement to "Alternative".
+
+Automatically Link Existing Account::
+  Automatically link brokered identities without any validation with this authenticator. This is useful in an intranet environment of multiple user databases each with overlapping usernames/email addresses, but different passwords and you want to allow users to use any password without having to validate.  This is only reasonable if you manage all internal databases, and usernames/email addresses from one database matching those in another database belong to the same person.  This authenticator would be dangerous in a generic environment where users can register themselves using arbitrary usernames / email addresses.  One more time - don't use this unless registration of users is carefully curated and usernames/email addresses are assigned, not requested.  Set the authenticator requirement to "Alternative".
```

Comment:
Is it possible to convert the whole text starting with sentence "This authenticator would be dangerous in generic ..." to WARNING box? I think it can be done just by starting it with next paragraph prefixed with the "WARNING: " . Something like:

```
WARNING: This authenticator would be dangerous in generic ...
```